core/src/com/unciv/logic/map/TileInfo.kt

```diff
@@ -439,11 +439,15 @@ open class TileInfo {
 
     /**
      * Implementation of _`tileFilter`_
-     * @see <a href="https://github.com/yairm210/Unciv/wiki/uniques#user-content-tilefilter">tileFilter</a>
+     * @see <a href="https://github.com/yairm210/Unciv/wiki/Uniques#tilefilter">tileFilter</a>
      */
     fun matchesFilter(filter: String, civInfo: CivilizationInfo? = null): Boolean {
+        if (filter.startsWith('{')) // multiple types at once - AND logic. Looks like:"{Forest} {Hill}"
+            return filter.removePrefix("{").removeSuffix("}").split("} {")
```

Comment:
Mmmm, we still get into the same problem we get into with the unit filters, which is that in Chinese they don't have spaces between them.
Maybe we should have a splitFilter(filter:String):Sequence<String> function that can split both with and without spaces? I think that's the simplest solution